package.json

```diff
@@ -58,5 +59,10 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
+  },
+  "overrides": {
+    "eslint-config-starstuff": {
+      "eslint-plugin-prettier": "^5.0.0"
```

Comment:
You may like to, instead, release a new major version of starstuff that's Prettier 3 compatible.